charts/values.global.yaml

```diff
@@ -49,11 +49,11 @@ resources:
       memory: 16M
   bash:
     limits:
-      memory: 32M
+      memory: 1M
```

Comment:
I hope `touch` CLI does not use more than 1M.. but it's likely we see occasional OOM here, no? Is it worth a risk?